extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/discovery/SceneDiscoveryServiceManager.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.digitalstrom.internal.discovery;
+
+import java.util.HashMap;
+
+import org.eclipse.smarthome.binding.digitalstrom.handler.BridgeHandler;
+import org.eclipse.smarthome.binding.digitalstrom.handler.SceneHandler;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.listener.SceneStatusListener;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.scene.InternalScene;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SceneDiscoveryServiceManager} manages the different scene discovery services and informs them about new
+ * added scenes.
+ *
+ * @author Michael Ochel - Initial contribution
+ * @author Matthias Siegele - Initial contribution
+ */
+public class SceneDiscoveryServiceManager implements SceneStatusListener {
```

Comment:
I do not really see the benefit of having this class. Couldn't you simply deal with all scene types within the SceneDiscoveryService class itself? This would make it all much clearer as it would be in line with the other discovery services.